### ISSUE TITLE
Update development document

### ISSUE THREAD

**Issue by darrelmiller** [OAI/OpenAPI-Specification#1346]
- [x] Process of merging PRs.  Voting process for acceptance.
- [x] Define major vs minor change and votes required.
- [x] Define bar for contributor
- [x] Add reference to LF Code of Conduct. (@earth2marsh)
- [x] Create issue templates for issues and PRs.
- [x] Describe scope of new TDC once existing TDC becomes TSC. (@earth2marsh)
- [x] Process for triaging tickets.  Labels/Milestones (labels? Bugs, Features, Support)
- [x] Guidelines for closing tickets.  Meaning of closed tickets.
- [x] Guidelines for the process to update the development document once the initial version produced by this ticket is done.

**Comment by darrelmiller** [comment]
Will move Google doc with development guidelines and create contributing guidelines file. Also, update references to development guidelines file elsewhere.

**Comment by LasneF** [comment]
@darrelmiller , even if the list is relevant , should we split it into multiple ticket ?

**Comment by ralfhandl** [comment]
Looks like @lornajane addressed all tasks with the updated https://github.com/OAI/OpenAPI-Specification/blob/main/CONTRIBUTING.md.

Can we close this issue?

**Comment by lornajane** [comment]
I think we can, so I will!
